# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/base/nsDocumentViewer.cpp
# Commit: 4312b2b5dda8
# Full Hash: 4312b2b5dda8c1af7b56ac1c55ab762aff485a2c
# Author: Botond Ballo <botond@mozilla.com>
# Date: 2020-04-28 16:30:11
# Regressor Bug: 1556556
# File Overlap Count: 1
# Description:
#   Bug 1556556 - Move GetCallbackTransform() into a new ViewportUtils class. r=kats
#   
#   This function (and helper functions that wrap it) will be used extensively
#   throughout layout code, so keeping it in APZCCallbackHelper seems awkward.
#   
# ==============================================================================

diff -r d11dbf6403a5 -r 4312b2b5dda8 layout/base/nsDocumentViewer.cpp
--- a/layout/base/nsDocumentViewer.cpp	Tue Apr 28 01:34:48 2020 +0000
+++ b/layout/base/nsDocumentViewer.cpp	Tue Apr 28 01:35:05 2020 +0000
@@ -19,6 +19,7 @@
 #include "nsIContent.h"
 #include "nsIContentViewer.h"
 #include "nsIDocumentViewerPrint.h"
+#include "nsIScreen.h"
 #include "mozilla/dom/BrowsingContext.h"
 #include "mozilla/dom/BeforeUnloadEvent.h"
 #include "mozilla/dom/PopupBlocker.h"
